com.pilosa.client/src/main/java/com/pilosa/client/Internal.java

```diff
@@ -399,9 +403,9 @@ public int hashCode() {
       }
       int hash = 41;
       hash = (19 * hash) + getDescriptorForType().hashCode();
-      if (getBitsCount() > 0) {
-        hash = (37 * hash) + BITS_FIELD_NUMBER;
-        hash = (53 * hash) + getBitsList().hashCode();
+        if (getColumnsCount() > 0) {
```

Comment:
I know this is a bit of pain but I feel the singular form (Column) is a bit more natural in this entire PR.

Reply:
`Internal.java` is automatically generated by protobuf compiler, but I'll look out for `getColumnsCount` in manual written code